scss/components/_menu.scss

```diff
@@ -52,6 +52,11 @@ $menu-item-background-hover: $light-gray !default;
 
   li {
     @include disable-mouse-outline;
+
+    @if $global-flexbox {
```

Comment:
Yeah sitting flush together also would be good... I was wanting it to similar to float!